### PR TITLE
Docs: Specify correct fs type for mkfs on volume creation

### PR DESCRIPTION
**Pull request by Himura2la**
The default value is ext2, which it is not supported (anymore?).

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Error example</summary>
 failed to setup alloc: pre-run hook "csi_hook" failed: mounting volumes: node plugin returned an internal error, check the plugin allocation logs for more information: rpc error: code = Internal desc = rpc error: code = Internal desc = need resize check failed on devicePath /dev/rbd0 and staingPath /local/csi/staging/my-volume/rw-file-system-single-node-writer/XXXX-XXXX-XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX-XXXXXXXXXXXXXXXX-XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX, error: Could not parse fs info on given filesystem format: ext2. Supported fs types are: xfs, ext3, ext4 
</details>

### CODE REVIEW

**Comment by zdover23** [comment]
@Himura2la: Thanks for the update! You'll need to add a "Signed-off-by" line, as described here: https://docs.ceph.com/en/reef/start/documenting-ceph/#commit-the-change

If you need help with this, let me know.

**Comment by Himura2la** [comment]
Yep, I'm waiting until the repo is cloned to amend the commit made from Web, thanks for quick reply, I'll do it in a few minutes!

**Comment by zdover23** [comment]
> Yep, I'm waiting until the repo is cloned to amend the commit made from Web, thanks for quick reply, I'll do it in a few minutes!

Roger that. Good luck!

**Comment by zdover23** [comment]
@neha-ojha: Can you give me technical confirmation that ext4 is the correct fs type?

**Comment by Himura2la** [comment]
By the way, we tried `xfs`, and it fails with other message, so for us, only `ext4` works

File: doc/rbd/rbd-nomad.rst [hunk 1]
```diff
@@ -372,6 +372,7 @@ using the newly created nomad user id and cephx key::
       clusterID = "b9127830-b0cc-4e34-aa47-9d1a2e9949a8"
       pool = "nomad"
       imageFeatures = "layering"
+      mkfsOptions = "-t ext4"
```
[the remark this answers is not in the document]
Comment (zdover23) [reply doc/rbd/rbd-nomad.rst:375]: Ilya's word is the gold standard. I'll merge and backport this now.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/55282 - Reef backport
https://github.com/ceph/ceph/pull/55283 - Quincy backport

**Pull request merged**